src/components/PathRouter.js

```diff
@@ -6,29 +6,33 @@ import LandingPage from "./pages/LandingPage";
 import UserProfile from "./pages/UserProfile";
 import EditItemDetailsPage from "./pages/EditItemDetailsPage";
 import NotFoundPage from "./pages/NotFoundPage";
+import ScrollToTop from "./ScrollToTop";
 
 const PathRouter = () => {
   return (
-    <Switch>
-      <Route exact path="/">
-        <LandingPage />
-      </Route>
-      <Route exact path="/items">
-        <ItemResults />
-      </Route>
-      <Route exact path="/items/item=:id">
-        <ItemPage />
-      </Route>
-      <Route exact path="/profile">
-        <UserProfile />
-      </Route>
-      <Route path="/items/item=:id/edit">
-        <EditItemDetailsPage />
-      </Route>
-      <Route>
-        <NotFoundPage />
-      </Route>
-    </Switch>
+    <React.Fragment>
+      <ScrollToTop />
```

Comment:
does this basically add a scroll bar in case the page gets very long? what does it actually do?

Reply:
So if you select an itemPreview near the bottom of the page, when you get taken to the itemsDetail page, it will scroll to the top of the page. Before, your position on the itemsDetail page would be the same as before the redirect